src/Sulu/Bundle/AdminBundle/Command/UpdateBuildCommand.php

```diff
@@ -252,16 +254,17 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         return 0;
     }
 
-    private function getLocalFile(string $path)
+    private function getLocalFile(string $path): string
     {
-        if (!\file_exists($this->projectDir . $path)) {
+        $content = \file_get_contents($this->projectDir . $path);
+        if (!\is_string($content)) {
             return '';
         }
 
-        return \file_get_contents($this->projectDir . $path);
+        return $content;
     }
```

Comment:
Try reading the file and when it fails return the default `''` technically a BC break because the file could exist and you just don't have read permissons then it would return an empty string which used to be false.

Reply:
No this behaviour should not be changed. The implementation with return empty string if not exist is expected in the upgrade process.

Reply:
PS: type change pull requests should not change any logic ;)

Reply:
Yeah this escalated a little. But I'll revert it.